torch/csrc/distributed/c10d/init.cpp

```diff
@@ -1187,7 +1187,13 @@ that adds a prefix to each key inserted to the store.
 
           .def(
               "_allgather_base",
-              &::c10d::ProcessGroup::_allgather_base,
+              [](const c10::intrusive_ptr<::c10d::ProcessGroup>& self,
+                 at::Tensor& output_tensor,
+                 at::Tensor& input_tensor,
```

Comment:
i think this can be const &

Reply:
I had a look at `init.cpp`, it seems the convention has not been using const. Not sure why. Can go with convention I guess.